### ISSUE TITLE
Bump version of openhtmltopdf to at least 0.0.1-RC18

### ISSUE THREAD

**Issue by gunjam** [dwp/ms-html-to-pdfa#1]
As of 0.0.1-RC18 openhtmltopdf has added accessibilty features such as titles, language and description meta data as well as tagged content. This brings it into compliance with the PDF/UA, Section 508 and WCAG 2 accessibility standards.

These added features would be greatly appreciated so we can produce accessible documents for our users via this service.

Thanks!

**Comment by HealthPDU** [comment]
Thanks for raising this; you're absolutely right that we should be supporting PDF/UA.  We will look into extending the api to support all (current) standards up to and including PDF/UA.

Many Thanks,

Chris

**Comment by HealthPDU** [comment]
The PDF/UA functionality is now covered in v1.5.0 and extends the current set of document conformance levels to include the PDF/UA format for accessibility.  The `accessible-test.html` test file (src/test/resources) was tested using Adobe Pro with the following outputs to confirm UA.

[Acrobat Accessibility Report.pdf](https://github.com/dwp/ms-html-to-pdfa/files/3436233/Acrobat.Accessibility.Report.pdf)

Thanks for raising this issue against the project 👍, I hope this release is useful.

Many Thanks,

Chris
